iidm/iidm-geodata/src/main/java/com/powsybl/iidm/geodata/dto/GeoShape.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.geodata.dto;
```

Comment:
For consistency, can you rename this package (and the corresponding test one) in `elements` instead of `dto`?